### PR TITLE
UserList added

### PR DESCRIPTION
**Pull request by tariqjavid**
# Description

It is list of users that  is display, i have  created there  files, UserListComponent.js, UserListStyle.css and UserList.stories.js 
#How you tested?  I have tested using storybook.

### CODE REVIEW

**Review by bytewiz — CHANGES_REQUESTED** [review]
The issue is not linked to the PR, please check the guide how to.

Also there is not explanation of the PR and none of the boxes are checked off.

Will review the code once that is done 

@tariqjavid

File: src/server/api/controllers/messages-controller.js [hunk 1]
```diff
@@ -61,4 +61,4 @@ module.exports = {
   deleteMessage,
   createMessage,
   editMessage,
-};
\ No newline at end of file
+};
```
Comment (benna100) [inline src/server/api/controllers/messages-controller.js:64]: Why are you changing this file?

File: src/server/api/routes/messages-router.js [hunk 2]
```diff
@@ -178,4 +178,4 @@ router.delete('/:id', (req, res) => {
     .catch((error) => console.log(error));
 });
 
-module.exports = router;
\ No newline at end of file
+module.exports = router;
```
Comment (benna100) [inline src/server/api/routes/messages-router.js:181]: And why this file?

File: src/client/components/UsersListComponent/UsersList.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './UsersListStyle.css';
+
+export default function Userslist(props) {
```
Comment (benna100) [inline src/client/components/UsersListComponent/UsersList.js:5]: This component does too much. It should only be the UserList. You have also added the h1, the input etc. Only add the ul and li as part of this component. Have you talked to Deepthi about how to get users? https://github.com/HackYourFuture-CPH/chattie/issues/29

**Review by benna100 — CHANGES_REQUESTED** [review]

File: src/server/api/controllers/messages-controller.js (hunk 1 above)
Comment (tariqjavid) [reply src/server/api/controllers/messages-controller.js:64]: i don't  know  how is changed

Comment (tariqjavid) [reply src/server/api/controllers/messages-controller.js:64]: i think, i was  reveiwing projects and might be possible  it is presses keyboard key

**Review by benna100 — APPROVED** [review]
Looks good Tariq, well done :)

**Comment by benna100** [comment]
Remember to update the story in the project board!

**Pull request merged**